token/program/src/instruction.rs

```diff
@@ -226,7 +226,7 @@ impl TokenInstruction {
         }
         Ok(match input[0] {
             0 => {
-                if input.len() < size_of::<u8>() + size_of::<u8>() + size_of::<bool>() {
+                if input.len() < size_of::<u8>() + size_of::<Pubkey>() + size_of::<bool>() {
```

Comment:
This math seems sketchy.  But I figure it'll get cleaned up by @jackcmay as he works through https://github.com/solana-labs/solana-program-library/issues/315